packages/flutter/lib/src/material/data_table.dart

```diff
@@ -1138,7 +1153,10 @@ class DataTable extends StatelessWidget {
     }
     else {
       final DataColumn sortColumn = columns[sortColumnIndex!];
+      // the intuitive column index, e.g. columns[0] is actually column 1 for users
```

Comment:
nit: ```suggestion
      // The intuitive column index, e.g. columns[0] is actually column 1 for users
```

Reply:
Thanks! Done. Migrating to https://github.com/flutter/flutter/pull/116750 due to merge conflict.

Reply:
Sg! Can this be closed?